frontend/src/components/Masthead/Navigation.tsx

```diff
@@ -0,0 +1,53 @@
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+
+type NavItem = {
+  name: string;
+  href: string;
+};
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Sorry, it's my fault that I didn't add it to design but let's add border-bottom on active tab: 
![image](https://user-images.githubusercontent.com/33123587/113544187-1c9dff80-95e8-11eb-82e9-94b971287982.png)

Reply:
added